src/lua/utils.c

```diff
@@ -120,6 +122,12 @@ luaL_pushuuidstr(struct lua_State *L, const struct tt_uuid *uuid)
 	lua_pushlstring(L, str, UUID_STR_LEN);
 }
 
+struct datetime *
+luaL_pushdatetime(struct lua_State *L)
```

Comment:
It is never used. At least in this commit. Please, remove.

Reply:
It will become useful in the messagepack patch. I'll move this code with definition to the different patch.

Reply:
Moved to the different patch of series.